### PR TITLE
removes collision from beds

### PR DESCRIPTION
**Pull request by tmtmtl30**
it's a bed, you should be able to move over it i think. because it's a fucking bed

### CODE REVIEW

File: Resources/Prototypes/Entities/Constructible/Furniture/beds.yml [hunk 1]
```diff
@@ -11,16 +11,7 @@
...
-      - Impassable
-      - MobImpassable
-      - VaultImpassable
-      - SmallImpassable
-      layer:
-      - Opaque
-      - MobImpassable
-      - VaultImpassable
-      - SmallImpassable
+      layer: [ Passable ]
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Constructible/Furniture/beds.yml:14]: So beds can now go through walls?

Comment (tmtmtl30) [reply Resources/Prototypes/Entities/Constructible/Furniture/beds.yml:14]: Alright. I was taking from #2916 -- should I re-add the `mask`, and remove the `layer` entirely?

Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Constructible/Furniture/beds.yml:14]: And I left the same comment on that pr lol.

Surely it shouldn't have no collision at all. I assume you know why mask and layer are split?

Comment (tmtmtl30) [reply Resources/Prototypes/Entities/Constructible/Furniture/beds.yml:14]: Layer represents the properties of the object (i.e. it is impassable / opaque / etc.), mask represents the layers the object will collide _with_ -- a bed is not impassable (not on the impassable layer), but it should collide with things that are (have impassable in its mask).

Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Constructible/Furniture/beds.yml:14]: Yeah, it's copied from godot.

In this case you'd probably reduce the layers, though the mask probably still needs Impassable.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**